tools/clang/unittests/HLSL/PixTest.cpp

```diff
@@ -3452,14 +3356,10 @@ void RaygenInternalName()
     if (findAlloca != metaDataKeyToValue.allocaDefinitions.end()) {
       if (findAlloca->second.count == 4) {
         RayPayloadElementCoverage[write.second.offset] = true;
-      } else if (findAlloca->second.count == 8) {
-        RayDescElementCoverage[write.second.offset] = true;
       }
     }
   }
   // Check that coverage for every element was emitted:
   for (auto const &b : RayPayloadElementCoverage)
     VERIFY_IS_TRUE(b);
-  for (auto const &b : RayDescElementCoverage)
-    VERIFY_IS_TRUE(b);
```

Comment:
I'm trusting the PIX test changes are all correct, it might be nice to get @jeffnn to say so in this review though.

Reply:
Yes, I confirmed with Jeff before I made the changes to the PIX tests.